src/client.js

```diff
@@ -2071,6 +2071,28 @@ MatrixClient.prototype.getEventTimeline = function(timelineSet, eventId) {
     return promise;
 };
 
+/**
+ * Get an event in a room by its event id.
+ *
+ * This blindly calls the /rooms/$roomId/event/$eventId endpoint.
+ *
+ * @param {string} roomId  The ID of the room to look in
+ * @param {string} eventId  The ID of the event to look for
+ *
+ * @return {Promise} Resolves to an object containing the event.
+ */
+MatrixClient.prototype.fetchRoomEvent = function(roomId, eventId) {
```

Comment:
Should this go in `MatrixBaseApis` (the base class of `MatrixClient`, so access wouldn't change) rather? That's where most of the pure network calls go, and as it doesn't return a `MatrixEvent` like most of the code in `MatrixClient`, but pure JSON instead, it feels like a bitter fit.

Reply:
Sure thing, I was mainly emulating some of the other calls we make that directly make calls. How do I transform a pure json (that is in standard C-S event format) to a MatrixEvent?

Reply:
Oh, ``new MatrixEvent({eventcontents})``

Reply:
Err, nevermind. I've just moved the func to base-apis.js